docs/CODE_GUIDELINES.md

```diff
@@ -660,21 +738,21 @@ A class with any virtual functions should have a destructor that is either publi
 For lines up to [line length](#line-length) everything stays on one line, excluding the braces which must be on the following lines.
 
 ```cpp
-MyClass::CMyClass(bool bBoolArg, int iIntegerArg) : m_bArg(bBoolArg), m_iArg(iIntegerArg)
+MyClass::CMyClass(bool argOne, int argTwo) : m_argOne(argOne), m_argTwo(argTwo)
 {
 }
 ```
 
 For longer lines, break before colon and after comma.
```

Comment:
For longer lines, insert a line break before the colon and/or after the comma.

_(assuming it's not intended that line breaks are **always** inserted in both places for longer lines)_